docs/index.rst

```diff
@@ -27,28 +27,35 @@ Python
 
 Python is a powerful programming language that allows concise expressions of network
 algorithms.  Python has a vibrant and growing ecosystem of packages that
-GraSPy uses to provide more features such as numerical linear algebra and
-plotting.  In order to make the most out of GraSPy you will want to know how
+graspologic uses to provide more features such as numerical linear algebra and
+plotting.  In order to make the most out of graspologic you will want to know how
 to write basic programs in Python.  Among the many guides to Python, we
 recommend the `Python documentation <https://docs.python.org/3/>`_.
 
 Free software
 -------------
 
-GraSPy is free software; you can redistribute it and/or modify it under the
-terms of the :doc:`Apache-2.0 </license>`.  We welcome contributions.
-Join us on `GitHub <https://github.com/neurodata/graspy>`_.
+graspologic is free software; you can redistribute it and/or modify it under the
+terms of the :doc:`MIT </license>`.  We welcome contributions.
+Join us on `GitHub <https://github.com/microsoft/graspologic>`_.
 
 History
 -------
 
-GraSPy was born in September 2018. The original version was designed and written by Jaewon Chung, 
-Benjamin Pedigo, and Eric Bridgeford.
+``graspologic`` first released in September 2020, but it got its start as a pair of Python libraries
```

Comment:
Please check me on this wordmaking here, feel free to add any changes you'd like to see or reword it all to make it flow differently, etc